rvv-intrinsic-generator/rvv_intrinsic_gen/main.py

```diff
@@ -168,6 +168,7 @@ class GenTypes:
     print("Skipping default RVV instructions (--skip-default-inst)")
   if vendor_gen is not None:
     vendor_gen(g)
+  g.post_gen()
```

Comment:
To be consistent, should have this post_gen function called under the document paths too.

Reply:
Sure, missed the doc part.